src/main/java/mekanism/common/item/gear/ItemJetpack.java

```diff
@@ -128,6 +143,31 @@ public int getDefaultTooltipHideFlags(@Nonnull ItemStack stack) {
         return super.getDefaultTooltipHideFlags(stack);
     }
 
+    /**
+     * Gets the first found jetpack from an entity, if one is worn. <br>
+     * If Curios is loaded, the curio slots will be checked as well.
+     *
+     * @param entity the entity on which to look for the jetpack
+     * @return the jetpack stack if present, otherwise an empty stack
+     */
+    @Nonnull
+    public static ItemStack getJetpack(final LivingEntity entity) {
+        final ItemStack chest = entity.getItemBySlot(EquipmentSlot.CHEST);
+        if (chest.getItem() instanceof ItemJetpack) return chest;
```

Comment:
This should have brackets and then to keep it shorter you can just do `} else if (Mekanism.hooks.CuriosLoaded) {`